schema_utils.py

```diff
@@ -169,6 +176,18 @@ class Normalizers(object):
 
     @classmethod
     def get(cls, normalizer_id):
+        """Gets the value of the normalizer_id from the specified object.
+
+        Args:
+            normalizer_id: str. The name of the normalizer method that
+                should be retrieved.
+
+        Returns:
+            *. The value of the normalizer_id that was passed by parameter.
```

Comment:
The normalizer method corresponding to the given normalizer_id.

Also what's the typeinfo? It's not *, that's too broad. I think it's function (or similar)?